github/repos_commits_test.go

```diff
@@ -72,6 +72,7 @@ func TestRepositoriesService_GetCommit(t *testing.T) {
 		      "status": "s",
 		      "raw_url": "r",
 		      "blob_url": "b",
+		      "contents_url": "c",
 		      "patch": "p"
```

Comment:
Minor and optional, but it could be nice to re-arrange these last 4 fields to match the same order as in `CommitFile`. "patch" would go below "status", and "raw_url" swapped with "blob_url".